cumulusci/cumulusci.yml

```diff
@@ -866,6 +890,7 @@ flows:
                 task: snapshot_changes
 
     install_prod:
+        group: Install / Uninstall
```

Comment:
install_prod belongs in the Org Setup group because it does the full setup including dependencies, installation, and post-install config. It should have been called prod_org, release_org, or customer_org